pkg/fakerp/fakerp.go

```diff
@@ -53,6 +54,8 @@ func GetDeployer(log *logrus.Entry, cs *api.OpenShiftManagedCluster, config *api
 }
 
 func createOrUpdate(ctx context.Context, log *logrus.Entry, cs, oldCs *api.OpenShiftManagedCluster, config *api.PluginConfig, isAdmin bool) (*api.OpenShiftManagedCluster, error) {
+	publicSubdomain := ""
```

Comment:
Why do you initialize them outside? can we just do:
`publicSubdomain := fmt.Sprintf("%s.%s", rDomain, os.Getenv("DNS_DOMAIN"))` ?

Reply:
Or initialize them before use itself.

Reply:
I was using this to move the DNS setup to the end of the file after cluster creation.  This was the if statement logic.  `if publicSubdomain != "" { // do dns create`.  Since moving the DNS to the end failed with the error from above I moved it back.  I think I can take your suggestion and let the follow up PR move the DNS.